qcelemental/models/procedures.py

```diff
@@ -0,0 +1,119 @@
+from pydantic import BaseModel, constr
+from .common_models import Model, DriverEnum, \
```

Comment:
Canonical way to do this is with parenthesis `from .common_models import (Model, DriverEnum ...)`.